packages/js/src/ui/components/Header/StatusOptions.tsx

```diff
@@ -16,33 +15,24 @@ export const StatusOptions = (props: { setFeedOptions: (options: FetchFeedArgs)
     <>
       <StatusItem
         label={DropdownStatus.UnreadRead}
-        /**
-         * TODO: Implement setFeedOptions and isSelected after Filter is implemented
-         */
         onClick={() => {
-          props.setFeedOptions({ status: NotificationStatus.UNREAD });
+          props.setFeedOptions({ archived: false });
         }}
         isSelected={true}
         icon={Inbox}
       />
       <StatusItem
         label={DropdownStatus.Unread}
         onClick={() => {
-          /**
-           * TODO: Implement setFeedOptions after Filter is implemented
-           */
-          props.setFeedOptions({ status: NotificationStatus.UNSEEN });
+          props.setFeedOptions({ read: false, archived: false });
         }}
         isSelected={false}
         icon={Unread}
       />
       <StatusItem
         label={DropdownStatus.Archived}
         onClick={() => {
-          /**
-           * TODO: Implement setFeedOptions after Filter is implemented
-           */
-          props.setFeedOptions({ status: NotificationStatus.SEEN });
+          props.setFeedOptions({ archived: true, read: true });
```

Comment:
✏ suggestion: when `archived: true` then `read: true` is always true. You can check the logic and the note in the `libs/dal/src/repositories/message/message.repository.ts:534`